src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -334,6 +334,31 @@ public static void ACos_Advanced(double real, double imaginary, double expectedR
             VerifyRealImaginaryProperties(result, expectedReal, expectedImaginary);
         }
 
+        public static IEnumerable<object[]> ACos_Legacy_TestData ()
+        {
+            // Boundary values
+            yield return new object[] { double.MaxValue, 0, double.NaN, double.NaN };
+            yield return new object[] { double.MinValue, 0, double.NaN, double.NaN };
+
+            // Invalid values
+            foreach (double invalidReal in s_invalidDoubleValues) {
```

Comment:
I'm really sorry that you guys have to find and call out this stuff. VS's formatting defaults are set for other projects that I work on, so on my side I have to remember to go and adjust the formatting of every line when I work on your project, and sometimes I miss some. I don't know of any easy way to change VS's formatting defaults on a per-project basis; do you?